### PR TITLE
New package: Hashicorp.Terraform.RC version 1.8.0-rc1

### PR DESCRIPTION
**Pull request by kdpuvvadi**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/146625)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335226
> 2024-03-26 23:03:56.573 [CLI ] ShellExecute installer failed: 127
> 2024-03-26 23:03:28.573 [CLI ] ShellExecute installer failed: 127

(Automated response - build 876.)

**Comment by kdpuvvadi** [comment]
What seems to be the problem here? i'm not sure about the error message

File: manifests/h/Hashicorp/Terraform/RC/1.8.0-rc1/Hashicorp.Terraform.RC.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Hashicorp.Terraform.RC
+PackageVersion: 1.8.0-rc1
+InstallerType: zip
+NestedInstallerType: exe
```
Comment (stephengillie) [inline manifests/h/Hashicorp/Terraform/RC/1.8.0-rc1/Hashicorp.Terraform.RC.installer.yaml:7]:
```suggestion
NestedInstallerType: portable
```

**Comment by stephengillie** [comment]
Verify:

**Review by stephengillie — APPROVED** [review]

**Pull request merged**